clusterctl/examples/google/provider-components.yaml.template

```diff
@@ -252,6 +252,12 @@ data:
           api:
             advertiseAddress: ${PUBLICIP}
             bindPort: ${PORT}
+          etcd:
```

Comment:
Please update clusterctl/examples/vsphere/provider-components.yaml.template as well.

Reply:
There is no analogous MasterConfiguration there. For vSphere, this is configured in cloud/vshpere/templates.go (hardcoded in the machine controller image). In this PR I edit this file as well. Long term I think it would be useful to have a consistency, although I don't have a full context - it was probably implemented differently for a reason.